File: Squad 02 - Estruturas de Árvore/Thaimara_Leitao_da_Silva/atividade2.py
# Função que recebe um array e devolve a soma dos elementos
def soma_array(ar):

    # A gente cria uma variável para guardar o resultado da soma.
    # Começa em 0 porque ainda não somamos nada.
    soma = 0

    # O for percorre o array inteiro.
    # A cada repetição, o numero recebe um valor do array.
    for numero in ar:

        # Pegamos o valor que já estava acumulado
        # e adicionamos o número atual.
        soma = soma + numero

    # Depois que o for termina
    # retornamos o valor final acumulado.
    return soma


# Lê o tamanho do array
n = int(input())

# Lê os números digitados e transforma em lista
ar = list(map(int, input().split()))

# Chama a função e dá o resultado
print(soma_array(ar))

File: Squad 02 - Estruturas de Árvore/Maicon Douglas Alves de Oliveira/atividade2.ts
function somaArray(arr: number[]): number {
    let resultado: number = 0;
    if(arr.length >= 0 && arr.length <= 1000){

        for(let i = 0; i < arr.length; i++) {

            let arrayTemp: number = arr[i];

            resultado += arrayTemp;
        
        }
        
    }
    return resultado;
}

somaArray([1, 2, 3, 4, 5]);
